### DISCUSSION TITLE
urllib3.exceptions.NameResolutionError: <urllib3.connection.HTTPSConnection object at 0x7f77c258b320>: Failed to resolve 'openaipublic.blob.core.windows.net' ([Errno -2] Name or service not known)

### DISCUSSION THREAD

**Discussion by zhengjianping1010** [langgenius/dify#11307] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

当我使用Dify中的OpenAI模型时，出现下面的问题：
urllib3.exceptions.NameResolutionError: <urllib3.connection.HTTPSConnection object at 0x7f77c258b320>: Failed to resolve 'openaipublic.blob.core.windows.net' ([Errno -2] Name or service not known)

**Comment by randolph555** [comment]
有解决方案了吗
